modelexpress_client/python/modelexpress/load_strategy/trainer_pull_strategy.py

```
@@ -0,0 +1,234 @@
+        if not ctx.accelerator_backend.supports_rdma_p2p():
+            return False
+        if ctx.mx_client is None:
+            return False
+        return True
+
+    def load(self, result: LoadResult, ctx: LoadContext) -> LoadResult:
+        result = _as_load_result(result)
+
+        try:
+            table = self._fetch_table(ctx)
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)